stingray/lightcurve.py

```diff
@@ -269,12 +373,13 @@ def shift(self, time_shift):
 
         """
         new_lc = Lightcurve(self.time + time_shift, self.counts,
-                            gti=self.gti + time_shift, mjdref=self.mjdref)
+                            err=self.counts_err,
+                            gti=self.gti + time_shift, mjdref=self.mjdref,
+                            dt=self.dt, err_dist=self.err_dist,
+                            skip_checks=True)
         new_lc.countrate = self.countrate
         new_lc.counts = self.counts
         new_lc.counts_err = self.counts_err
```

Comment:
If you put in `self.counts_err` etc in above, do we still need these three lines?

Reply:
Nope, it was a leftover. Good catch :)